src/main/java/com/everis/boundary/CarResources.java

```diff
@@ -0,0 +1,88 @@
+package com.everis.boundary;
+
+import java.util.List;
+
+import javax.ejb.EJB;
+import javax.ws.rs.*;
+import javax.ws.rs.core.*;
+import javax.ws.rs.core.Response.*;
+
+import org.apache.log4j.Logger;
+import com.everis.boundary.CarResources;
+import com.everis.control.CarService;
+import com.everis.entity.Car;
+
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+
+
+@Path("cars")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+public class CarResources {
+
+    
+    CarService carService;
```

Comment:
Maybe missing the initialization or injection of carService ? Please, review it.